packages/react-scripts/README.md

```diff
@@ -94,5 +79,32 @@ If you decide to opt out, Sass files will not be treated as CSS Modules by defau
 
 ## Releasing a new version of `backpack-react-scripts`
 
-Please refer to the [CONTRIBUTING.md](./../../CONTRIBUTING.md) in the root of this repo. All other packages have had 
-their `package.json`'s modified to be private therefore they won't be published to the npm registry.
+1. To publish a new version of `backpack-react-scripts`, run the following command:
+    
+    ```
+    npm run publish -- --scope backpack-react-scripts
+    ```
+
+2. You will be prompted to select a new semver version (MAJOR, MINOR, PATCH). Use the [CHANGELOG.md](./CHANGELOG.md) to decide on the nature of the changes since the last release.
+    * If you want to be extra careful, you can publish a prerelease by running this instead:
+    
+    ```
+    npm run publish -- --scope backpack-react-scripts --canary
+    ```
+
+4. Update the [CHANGELOG.md](./CHANGELOG.md) with the new version, taking care to follow the format of previous releases.
```

Comment:
It'll save precious seconds from re-numbering them if you change them at a later date!